pkgs/by-name/ro/roomarranger/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  at-spi2-atk,
+  autoPatchelfHook,
+  cairo,
+  cups,
+  fetchurl,
+  gdk-pixbuf,
+  gtk3,
+  lib,
+  libGL,
+  qt6,
+  stdenv,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "roomarranger";
+  version = "10.0.1";
+
+  src = fetchurl {
+    url = "https://f000.backblazeb2.com/file/rooarr/rooarr1001-linux64.tar.gz";
```

Comment:
This URL looks very bizarre but it is linked from the https://www.roomarranger.com/ homepage.